### DISCUSSION TITLE
Progress on Currency Converter Coding suggestions would be appreciated and will speed things along.

### DISCUSSION THREAD

**Discussion by gh44922023** [gh44922023/Tafe230221#10] [General]
Just checking on the progress of "Currency Converter" Development.
Any issues to report?
Initial struggles with Visual Studio - never used it before - but am working through them - 
am at the stage where extracting data from combobox selection occurs. and assigning it to label contents to show as output.

Any screenshots to share?
see attached files
Blank boxes are stackpanels holding out put labels, once I have sorted out coding side. Any coding suggestions for visual studio would be appreciated and may speed things along.

be advised the 4 files are not in the repos, Tafe230221.

![convo09sh2](https://github.com/gh44922023/Tafe230221/assets/142568314/c3f0cd3b-55a6-47e0-b278-4ac4a24d869f)
![convo9sh1](https://github.com/gh44922023/Tafe230221/assets/142568314/beac5baf-86d9-456a-8a3d-1f09c007a260)
[Currency Converter.xaml.cs.txt](https://github.com/gh44922023/Tafe230221/files/12482994/Currency.Converter.xaml.cs.txt)
[Currency Converter.xaml.txt](https://github.com/gh44922023/Tafe230221/files/12482995/Currency.Converter.xaml.txt)

**Comment by gh44922023** [comment]
With assistance from Jeff, I am happy to report the Currency Converter is ready to merge with the development branch.
I plan to do this after Brodie has completed his merge.  Please advise if any issues with this.
![currConvWorking](https://github.com/gh44922023/Tafe230221/assets/142568314/3fbe1d5d-4725-498d-8b39-0887b351ad37)

**Comment by jeffrey-tafe** [comment]
Looks good. @BrodieBrown1998 It looks like you and @gh44922023 have both built a menu page? Is tha right?
